python/tvm/relay/frontend/common.py

```diff
@@ -41,6 +41,20 @@ class StrAttrsDict(object):
     def __init__(self, attrs):
         self.attrs = attrs
 
+    def is_attr_present(self, key):
```

Comment:
let's think about the name, shall we just name it `has_attr`?

Reply:
I don't have a problem with `has_attr`. i can submit a change with that name.